### PR TITLE
jNoSQL Jakarta data MongoContainer and test

### PR DESCRIPTION
**Pull request by mswatosh**
Adding the MongoContainer to test jNoSQL with, and a basic test using jNoSQL as a Jakarta Data implementation.

### CODE REVIEW

**Comment by helyarp** [comment]
@mswatosh `CWWKDMessages.nlsprops` does not have new or changed strings. Unless there are message strings to review, pls add the **ID review not needed** label.

**Comment by mswatosh** [comment]
#build

File: dev/io.openliberty.data.internal_fat_nosql/bnd.bnd [hunk 1]
```diff
@@ -33,4 +33,4 @@ fat.project: true
 	io.openliberty.jakarta.servlet.6.0,\
 	io.openliberty.jakarta.transaction.2.0,\
 	io.openliberty.org.eclipse.microprofile.config.3.0,\
-	io.openliberty.org.eclipse.jnosql.1.0
```
Comment (KyleAure) [inline dev/io.openliberty.data.internal_fat_nosql/bnd.bnd:36]: You'll want to add the following line to this file:

fat.test.container.images: mongo:6.0.6

File: dev/io.openliberty.data.internal_fat_nosql/fat/src/test/jakarta/data/nosql/FATSuite.java [hunk 2]
```diff
@@ -26,6 +29,10 @@
                 DataNoSQLTest.class
 })
 public class FATSuite {
```
Comment (KyleAure) [inline dev/io.openliberty.data.internal_fat_nosql/fat/src/test/jakarta/data/nosql/FATSuite.java]: Add `extends TestContainerSuite` to make sure the remote docker host is set up before this bucket runs.

**Review by KyleAure — CHANGES_REQUESTED** [review]

**Review by KyleAure — APPROVED** [review]

**Comment by mswatosh** [comment]
#build

**Comment by mswatosh** [comment]
#libby

**Pull request merged**